src/Http/Environment.php

```diff
@@ -369,8 +369,7 @@ protected function detectForwardedHttps(): bool
             return true;
         }
 
-        // fall back to local setup
-        return $this->detectHttps();
+        return false;
```

Comment:
I'm thinking if we should rather default to `true` for HTTPS. In production it's certainly the most likely case, so it would be safer. Is there a specific reason why the default should be `false`?

Reply:
Just saw your comment in the previous discussion above.

> I just checked again and in a non-proxy setup, $_SERVER['HTTPS'] is undefined, if there's no HTTPS setup. IMHO the correct solution would be to simply return false instead of falling back to the local setup.

In the local setup, PHP gets the HTTPS status directly from the web server, so it can 100% rely on that. With a reverse proxy in front, the HTTPS status could really be both and PHP wouldn't know. It's a different situation and I don't think we can compare the two.

I'm still unsure if we should default to one value or if we should consider this case a misconfiguration and throw an exception. If we want to return a fallback default, I think that default should be `true` though.